internal/mbsc/helper.go

```diff
@@ -0,0 +1,30 @@
+package mbsc
+
+import (
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+)
+
+//go:generate mockgen -source=helper.go -package=mbsc -destination=mock_helper.go
+
+type Helper interface {
```

Comment:
What do you think about using `ModuleBuildSignConfigAPI` istead? and changing the file name to `mbsc.go`?
I know we are using `helper` for NMC but IMO it is not a great name. 

I am also using the same convention in the MIC package: https://github.com/ybettan/kernel-module-management/blob/37acf32101e6da5bcf71f49610d7e728880de930/internal/mic/mic.go#L19